components/sync15/src/sync15.udl

```diff
@@ -0,0 +1,19 @@
+// This file contains common sync15-related types which are used by various components.
```

Comment:
hrmph, `DeviceType` is actually defined in sync15-traits and re-exported from sync15, so having a sync15.udl seems wrong for this type.